src/gui/torrentmodel.cpp

```diff
@@ -258,6 +264,23 @@ bool TorrentModel::setData(const QModelIndex &index, const QVariant &value, int
     case TR_CATEGORY:
         torrent->setCategory(value.toString());
         break;
+    case TR_TAGS: {
```

Comment:
OK, I removed the indirection to setData for the tags code and left the old code alone.